### DISCUSSION TITLE
Tina Markdown Table Template

### DISCUSSION THREAD

**Discussion by isaaclombardssw** [tinacms/tinacms#4793] [Experimental]
Tina comes inbuilt with a way to manage tables in markdown – the `tinaTableTemplate` class!

While this feature has been around for some time, it's still experimental and has a few caveats to its usage.

This discussion is about suggestions or fixes to the tinaTableTemplate.

**Comment by JackDevAU** [comment]
Hey @isaaclombardssw 

We're currently working on a Plate table plugin, and it should be ready in the very near future!

You can check it out over on Plate @ https://platejs.org/docs/table

**Reply by 99ansh** [reply]
Hi @JackDevAU I had a similar ask [on discord](https://discord.com/channels/835168149439643678/1265875619577593907/1267617021768765522) 2 months back. Would it be possible to share a tentative timeline of feature availability? We heavily use tables for our site :)

Thanks a lot!

**Reply by wicksipedia** [reply]
Hey @99ansh this has now shipped in TinaCMS 2.3.0

**Comment by wicksipedia** [comment]
Closing discussion as feature has shipped in 2.3.0
